Rubberduck.Parsing/Symbols/ReferencedDeclarationsCollector.cs

```diff
@@ -90,11 +90,10 @@ public IEnumerable<Declaration> GetDeclarationsForReference(Reference reference)
         {
             var projectName = reference.Name;
             var path = reference.FullPath;
-
-            var projectQualifiedModuleName = new QualifiedModuleName(projectName, projectName);
+            var projectQualifiedModuleName = new QualifiedModuleName(projectName, path, projectName);
             var projectQualifiedMemberName = new QualifiedMemberName(projectQualifiedModuleName, projectName);
 
-            var projectDeclaration = new Declaration(projectQualifiedMemberName, null, null, projectName, false, false, Accessibility.Global, DeclarationType.Project);
+            var projectDeclaration = new ProjectDeclaration(projectQualifiedMemberName, projectName);
```

Comment:
**WHY** did I wait so long before seriously considering that inheritance hierarchy again?

Great job :+1: